protocol/contracts/Constants.sol

```diff
@@ -163,4 +180,28 @@ library Constants {
     function getDeployerLockupEnd() internal pure returns (uint256) {
         return DEPLOYER_LOCKUP_END;
     }
+
+    function getDaoAddress() internal pure returns (address) {
+        return DAO_ADDRESS;
+    }
+
+    function getGoldAddress() internal pure returns (address) {
+        return GOLD_ADDRESS;
+    }
+
+    function getHybridOraclePoolAddress() internal pure returns (address) {
+        return HYBRID_ORACLE_POOL_ADDRESS;
+    }
+
+    function get_SXAU_ESG_Addresses() internal pure returns (address, address) {
```

Comment:
The HybridOraclePool contains a collection of (Oracle/Pool) pairs, it makes sense to only return these as tuples IMO